src/components/Teaser/Teaser.tsx

```diff
@@ -31,16 +39,43 @@ export const Teaser: React.FC<Props> = (props) => {
                     </Text>
                 )}
             </div>
-            {subTitle && (
-                <Text inline context="accent" {...elem('subTitle', props)}>
-                    {subTitle}
-                </Text>
-            )}
-            {details && (
-                <Text inline context="muted">
-                    {details}
-                </Text>
-            )}
+            <div {...elem('line', props)}>
+                {subTitle && (
+                    <Text inline context="accent" {...elem('subTitle', props)}>
+                        {subTitle}
+                    </Text>
+                )}
+                {!subTitle && details && (
+                    <Text inline context="muted" {...elem('details', props)}>
+                        {details}
+                    </Text>
+                )}
+            </div>
+            <div {...elem('lastLine', props)}>
+                {subTitle && details && (
+                    <Text inline context="muted" {...elem('details', props)}>
+                        {details}
+                    </Text>
+                )}
+                {statuses && statuses.length > 0 && (
```

Comment:
How about extract wrapped `subTitle` and `details` into variables in order not to repeat same code?

```tsx
...

export const Teaser: React.FC<Props> = (props) => {
    const { title, subTitle, location, details, statuses, ...rest } = props;
// 👀 <---
    const subTitleElem = (
        <Text inline context="accent" {...elem('subTitle', props)}>
                {subTitle}
        </Text>
    );
// 👀 <---
    const detailsElem = (
        <Text inline context="muted" {...elem('details', props)}>
                {details}
        </Text>
    );

    return (
        <div {...rest} {...block(props)}>
            <div {...elem('line', props)}>
                <Text inline context="brand" {...elem('title', props)}>
                    {title}
                </Text>
                {location && (
                    <Text inline context="muted" {...elem('location', props)}>
                        {location}
                    </Text>
                )}
            </div>
            <div {...elem('line', props)}>
// 👀 <---
                {subTitle ? subTitleElem : }
                {!subTitle && details && detailsElem}
            </div>
            <div {...elem('line', props)}>
// 👀 <---
                {subTitle && details && detailsElem}
    ...
```

Reply:
we don't need subTitleElem because it is used once and I usually don't like to move render parts to variable or separate methods, moreover such small. But I don't mind to change :)

Reply:
No need to extract `subTitleElem indeed